src/app/services/submissions.service.ts

```diff
@@ -28,7 +28,12 @@ export class SubmissionsService {
   /**
    * Create new record.
    */
-  create(token, url, bodyData = {}) {
+  create(token, url, bodyData = {}, requestParam = {}) {
+    let requestOptions = new RequestOptions({
```

Comment:
`requestOptions` is not used in any other part of the service, correct? If it is not used, why do we want this? Sorry, if this doesn't make sense

Reply:
It is a mistake, that I was not using it. I fixed it now.